account/identity.go

```diff
@@ -202,7 +202,7 @@ func (m *GormIdentityRepository) Lookup(ctx context.Context, username, profileUR
 func (m *GormIdentityRepository) Save(ctx context.Context, model *Identity) error {
 	defer goa.MeasureSince([]string{"goa", "db", "identity", "save"}, time.Now())
 
-	obj, err := m.Load(ctx, model.ID)
+	_, err := m.Load(ctx, model.ID)
```

Comment:
Do we really need to load it with the new approach? What happen if you do m.db.Save(model) for unknown model? It should generate the same not-found error I guess?

Reply:
Why should we be concerned with the not-found error? to run update we need user token, and if we have the user token that means the account exist, hence found.

Reply:
Should be safe to remove that error check